src/settings.py

```diff
@@ -236,24 +236,57 @@
 DISABLED_ROLES = []
 
 NO_VICTIMS_MESSAGES = ("The body of a young penguin pet is found.",
+                       "Paw prints are found circling around the village.",
                        "A pool of blood and wolf paw prints are found.",
+                       "The body of a slain cat is found.",
+                       "Some house doors have been opened, but nothing has changed.",
+                       "A scent much like that of a wolf permeates the air.",
+                       "Half-buried wolf droppings are found.",
                        "Traces of wolf fur are found.")
 LYNCH_MESSAGES = ("The villagers, after much debate, finally decide on lynching \u0002{0}\u0002, who turned out to be... a{1} \u0002{2}\u0002.",
+                  "After a prolonged struggle, \u0002{0}\u0002 is forced to the gallows, and is discovered after death to be a{1} \u0002{2}\u0002.",
+                  "The villagers have chosen to hang \u0002{0}\u0002, but the rope stretches and breaks. However, the ensuing fall kills the \u0002{2}\u0002.",
+                  "The villagers, heavy with the pain of death, reluctantly lynch \u0002{0}\u0002, a{1} \u0002{2}\u0002.",
+                  "Compliant with the will of the village, the gallows prove effective in killing \u0002{0}\u0002, a{1} \u0002{2}\u0002.",
+                  "Galvanized by fear, the mob puts to death \u0002{0}\u0002. After inspection, they find that they have killed a{1} \u0002{2}\u0002.",
+                  "In a fit of hysteria, the villagers lynch \u0002{0}\u0002, killing a{1} \u0002{2}\u0002.",
+                  "Believing their fellow villager and friend to be dangerous, the mob puts \u0002{0}\u0002, a{1} \u0002{2}\u0002, to death.",
+                  "Deaf to the frenzied pleas of \u0002{0}\u0002, the village kills a{1} \u0002{2}\u0002.",
```

Comment:
It seems pretty obvious to anyone that knows the gameplay well, but I get your point- it might be confusing to some, especially new people. If English had a singular neutral third person pronoun, I could easily fix it, but... it'll be removed, I guess